components/GlobalStyles/typography.ts

```diff
@@ -0,0 +1,31 @@
+import { css } from "@emotion/react";
+
+export const typography = css`
+  @font-face {
+    font-family: "Halyard Display";
+    src: url("/fonts/HalyardDisplay-Regular.woff2") format("woff2"),
+      url("/fonts/HalyardDisplay-Regular.woff") format("woff");
+    font-weight: normal;
+    font-style: normal;
+    font-display: swap;
```

Comment:
just a tip (nothing to change), you can save on bundle size by only importing latin characters, like we do in Across https://github.com/across-protocol/frontend-v1/blob/65abd7772704a9ec243fd370f9e8e76322f0905b/src/components/GlobalStyles/GlobalStyles.tsx#L14